### PR TITLE
Automatically thank boosters

### PR DESCRIPTION
**Pull request by navarr**

### CODE REVIEW

File: src/Utility/NitroBoost.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const BotModule = require('../BotModule');
+
+const Message = {
```
Comment (McJeffr) [inline src/Utility/NitroBoost.js]: Variables shouldn't start with an uppercase letter.

Comment (navarr) [reply src/Utility/NitroBoost.js]: But we've been doing this all the time for things that we don't treat like variables (like BotModule)

Comment (McJeffr) [reply src/Utility/NitroBoost.js]: It's breaking the naming convention that you can see in commands such as support, whowas, playercount, mcstatus, fry, etc.

**Comment by navarr** [comment]
@McJeffr Do I have approval after that change?

**Review by McJeffr — APPROVED** [review]

**Pull request merged**